### ISSUE TITLE
Malli doesn't work with latest Cherry

### ISSUE THREAD

**Issue by ikitommi** [metosin/malli#1047]
```clojure
-           :cherry {:extra-deps {io.github.squint-cljs/cherry {:git/sha "0c4e0d1de534a4cf910ed7079638dbcfec8ec8b7"}}}
+           :cherry {:extra-deps {io.github.squint-cljs/cherry {:git/sha "3f78c7e03e1b940c0cbc05e8c5bc76f8295055cb"}}}
```

... fails with missing namespace aliases with SCI

**Comment by borkdude** [comment]
Should be fixed now with cherry master

**Comment by opqdonut** [comment]
this was fixed in 567a562a05192824da5a4f74f31fe806da8bfaeb
